### DISCUSSION TITLE
关于部署2.0.1版本

### DISCUSSION THREAD

**Discussion by Allen-zy** [apache/dolphinscheduler#7719] [Q&A] [answered]
在按照官方文档配置完成之后，DS集群的各个节点都能启动，登陆webUI也没问题，zookeeper注册信息也能查看到master,worker节点信息，但是在创建租户是报错，我的系统操作用户是root，wubo,都是超级管理员，同时wubo而言是hdfs的supergroup,但还是无法创建，通过查看日志，发现master日志信息如下：
![bd3813335b11b02f468f82bde866041](https://user-images.githubusercontent.com/53860974/147716787-fb6ea5e6-0706-4b24-837b-752e8bd014a2.png)
![fbf20ebbbbd985b47138a41a96b8550](https://user-images.githubusercontent.com/53860974/147716795-8266edd4-83ed-47bd-a2a1-3081516c4103.png)
大体意思是master 接受不到 worker心跳信息，但是各个节点的进程也好，zk的日志信息也好，都找不到，webUI还能再监控中心看到master和worker node 的信息，现在租户无法创建，无法测试其他功能，麻烦各位大佬给我分析下原因；
zk 版本 3.5.7，mysql 版本 5.7.16

**Comment by Tandoy** [comment, marked as answer]
Maybe you should check the logs of Apiserver

**Reply by Allen-zy** [reply]
谢谢你，通过API日志找到问题了， Permission denied: user=hdfs, access=WRITE, inode="/":wubo:supergroup:drwxr-xr-x

**Comment by davidzollo** [comment]
English is needed, thx
